Jellyfin.Server.Implementations/Item/BaseItemRepository.cs

```diff
@@ -99,19 +99,19 @@ public BaseItemRepository(
     }
 
     /// <inheritdoc />
-    public void DeleteItem(Guid id)
+    public void DeleteItem(Guid[] ids)
```

Comment:
This can be made to accept `IReadOnlyList<>` and then the call in LibraryManager doesn't need a `.ToArray()`

Maybe make this pass `params` as well so it isn't a breaking change for existing callers